partner_industry_secondary/migrations/11.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_res_partner_industry(cr):
+    cr.execute(
+        """
+        INSERT INTO res_partner_industry (name, active, create_uid,
+            create_date, write_uid, write_date, parent_id, %s)
+        SELECT name, TRUE as active, create_uid, create_date, write_uid,
+            write_date, parent_id, id as %s
+        FROM res_partner_sector
+        """ % (
+            openupgrade.get_legacy_name('old_id'),
+            openupgrade.get_legacy_name('old_id'),
+        )
+    )
+    cr.execute(
+        """
+        UPDATE res_partner_industry rpi2
+        SET parent_id = rpi.id
+        FROM res_partner_industry rpi
+        WHERE rpi2.parent_id = rpi.%s
+        """ % openupgrade.get_legacy_name('old_id')
+    )
+
+
+def update_res_partner_industry(cr):
+    cr.execute(
+        """
+        UPDATE res_partner rp
+        SET industry_id = rpi.id
+        FROM res_partner_industry rpi
+        WHERE rp.%s IS NOT NULL AND rp.%s = rpi.%s
+        """ % (
+            openupgrade.get_legacy_name('sector_id'),
+            openupgrade.get_legacy_name('sector_id'),
+            openupgrade.get_legacy_name('old_id'),
+        )
+    )
+
+
+def fill_res_partner_industry_secondary(cr):
+    cr.execute(
+        """
+        INSERT INTO res_partner_res_partner_industry_rel (res_partner_id,
+            res_partner_industry_id)
+        SELECT rel.res_partner_id, rpi.id as res_partner_industry_id
```

Comment:
No need of the `as res_partner_industry_id` part

Reply:
But also no need to take it out

Reply:
Yeah, but you improve a lot the size of the query (that is logged), the readability, etc. I'm telling you this also as a didactic exercise for you to know it.

Reply:
But this one is not logged :S

And the readability is subjective. I think with the "as xx" is more understandable.

Reply:
I don't agree, but I'm not going to force you on this, but please remove it in the other that you need an extra argument for formatting it.